include/glow/Backends/CompiledFunction.h

```diff
@@ -16,11 +16,25 @@
 #ifndef GLOW_BACKENDS_COMPILEDFUNCTION_H
 #define GLOW_BACKENDS_COMPILEDFUNCTION_H
 
+#include <glow/Graph/Nodes.h>
```

Comment:
Why do we need nodes.h?  Can we use a forward declaration?

Reply:
I may be going about it wrong, but it seems it needs to be there as part of the hash validity check on the unordered_map.